### ISSUE TITLE
Problem with sprites in android build.

### ISSUE THREAD

**Issue by HappySapeta** [keijiro/KinoBloom#13]
When used in a scene with sprites, screen shows random static dots and lines or a completely blank screen.
Could be a shader compatibility issue.

Thanks a lot for this free image effect. Great effort.
Please try to fix the above mentioned issue.
![screenshot_2018-10-16-01-16-16-206_com trinity project1 1](https://user-images.githubusercontent.com/35849508/46974302-4776ff00-d0e1-11e8-9b28-51e0f05bd9aa.png)

**Comment by keijiro** [comment]
I haven't maintained this repository for years. Please try [Post Processing Stack from Unity](https://assetstore.unity.com/packages/essentials/post-processing-stack-83912). It's better, faster and well-maintained.

**Comment by HappySapeta** [comment]
No, your shader works much better than unity's PPS on mobile, on 3D meshes
(PPS's 10 fps vs KinoBloom's 30+ fps). However, it messes up sprites for
some reason.
I really need this shader to work.
Thanks.
